moshi/src/main/java/com/squareup/moshi/JsonReader.java

```diff
@@ -214,6 +214,9 @@
   /** True to accept non-spec compliant JSON */
   private boolean lenient = false;
 
+  /** True to throw a {@link JsonDataException} on any attempt to call {@link #skipValue()}. */
+  private boolean skipForbidden = false;
```

Comment:
When I first read this, I thought it was a boolean to control skipping something classified as a forbidden value. Not forbidding the act of skipping values.